rspace-bench/src/test/scala/coop/rchain/rspace/bench/serialization/ExamplesSerializerBench.scala

```diff
@@ -11,6 +11,7 @@ import coop.rchain.rspace.bench.serialization._
 
```

Comment:
import on line 10 is not required

Reply:
good catch!
I will do that in a separate one if that's alright with you